src/lib/Commands.tsx

```
@@ -154,19 +149,17 @@ export function CommandsPalette({
     <CommandCenterDialog
       onClose={() => setPage(undefined)}
       ref={(ref) => (dialog = ref)}
-      class={
-        "relative mx-auto w-96 max-w-full transform flex-col overflow-hidden rounded-xl bg-white p-0 shadow-2xl ring-1 ring-black ring-opacity-5 transition-all backdrop:bg-white backdrop:bg-opacity-30 dark:bg-gray-900 dark:backdrop:bg-black dark:backdrop:bg-opacity-30 [&[open]]:flex"
-      }
+      class="relative mx-auto w-96 max-w-full transform flex-col overflow-hidden rounded-xl bg-white p-0 shadow-2xl ring-1 ring-black/5 transition-all backdrop:bg-white/30 open:flex dark:bg-gray-900 dark:backdrop:bg-black/30"
     >
       <div class="flex justify-end">
```

ESLint

[error] Unknown property 'class' found, use 'className' instead

(react/no-unknown-property)